tests/pytorch_tests/model_tests/feature_models/gptq_test.py

```diff
@@ -235,13 +236,92 @@ def compare(self, ptq_model, gptq_model, input_x=None, quantization_info=None):
         self.unit_test.assertTrue(all(w_diff), msg="GPTQ: some weights weren't updated")
 
 
-class SymGumbelLearnRateZeroTest(GPTQBaseTest):
+class UniformGumbelAccuracyTest(GPTQBaseTest):
+
+    def __init__(self, unit_test):
+        super().__init__(unit_test)
+        self.experimental = False
```

Comment:
Why by default we use the non-experimental facade? Also, please use experimental as an argument.

Reply:
Because experimental is not supported for Uniform (I get error for non support)